css/css-grid/masonry/tentative/intrinsic-sizing/masonry-intrinsic-sizing-cols-001-ref.html

```diff
@@ -26,9 +25,13 @@
 .fr grid {
   grid-template-columns: 1fr 2fr 1fr 1fr;
 }
-.mixed grid {
+.mix1 grid {
```

Comment:
I think we can spell out `mix1` and `mix2` with more descriptive names.

`mix1` could be `mix-fr-intrinsic`.
`mix2` could be `mix-explicit-fr-intrinsic`.